_rules/heading-descriptive-b49b2e.md

```diff
@@ -29,11 +29,11 @@ This rule applies to any element with the [semantic role][] of heading that is e
 
 ## Expectation 1
 
-Each target element which is [visible][] describes the topic or purpose of the first [palpable content][] which is non-[decorative][], [visible][], and after the target in tree order in the [flat tree][].
+For each target element, either it is not [visible][], or it describes the topic or purpose of the first [palpable content][] which is non-[decorative][], [visible][], and after it in tree order in the [flat tree][].
```

Comment:
I find this a little confusing. With "it" you mean the test target? I would appreciate something a little more explicit.